plugins/UM3NetworkPrinting/src/ClusterUM3OutputDevice.py

```diff
@@ -50,7 +48,7 @@ class ClusterUM3OutputDevice(BaseCuraConnectDevice):
 
     # This is a bit of a hack, as the notify can only use signals that are defined by the class that they are in.
     # Inheritance doesn't seem to work. Tying them together does work, but i'm open for better suggestions.
-    clusterPrintersChanged = pyqtSignal()
+    _clusterPrintersChanged = pyqtSignal()
```

Comment:
Not sure if this should be private, since QML can listen to it as well.

Reply:
_clusterPrintersChanged gets notified when things change, but it's only called in
`self.printersChanged.connect(self._clusterPrintersChanged)`
so the signal to be used externally should be printersChanged which is in a base class